### ISSUE TITLE
show label after chosing an autocomplete value (and field has still focus)

### ISSUE THREAD

**Issue by larsbo** [richardvenneman/floatl#5]
When you double click in an input field and choose a suggested value instead of typing the floating label stay hidden until the field lost focus.

It would make sense to show the label right after the completion of the field.

**Comment by larsbo** [comment]
Ok, this is a bit tricky because there is no "autosuggest chosen" event.
I found a trick at [stackoverflow](http://stackoverflow.com/a/33412873) that works at least for Firefox and Chrome (I didn't test any more yet).

My solution:

``` js
$('body').bind('input', function(event) {
    var input = $(event.target);
    if (input.hasClass('floatl__input')) {
        input.trigger('blur').trigger('focus');
    }
});
```

PS: It only works with binding to `input` - something like `$('body').bind('.floatl__input', ...`wouldn't work.
Triggering the `change`event was not sufficient strangely. So I had to trigger `blur`and `focus`.
